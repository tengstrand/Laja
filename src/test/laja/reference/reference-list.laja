
#macro three()
2##
#end

#write "target/test-output/reference-list.out"
  #set (list0 = [])
  #set (list1 = [ "first element of list1" ])
  #set (list2 = [ "middle", "begin {list2[0]} end", "Three" ])
  #set (one = "1")
Empty list = {list0}
The value of list1 is: {list1[0]}
list2[one] = {list2[one]}
list2[three()] = {list2[three()]}
#end