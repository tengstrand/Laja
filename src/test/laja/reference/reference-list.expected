Empty list = []
The value of list1 is: first element of list1
list2[one] = begin middle end
list2[three()] = Three
